#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("angular_threshold", double_t, 0, "angular threshold", 0.1, 0.0, 1.0)

exit (gen.generate (PACKAGE, "jsk_pcl_ros", "ParallelEdgeFinder"))
